index.html

```diff
@@ -54,7 +54,240 @@ <h2 class="contact-title">LET'S CONNECT</h2>
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/v.png" alt="V"
+                /></a>
+              </li>
+            </ul>
+            <button class="btn-default" type="button">Get my resume</button>
+          </div>
+          <ul class="skill-list">
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Languages</h3>
+                <button class="btn-chev" type="button">
+                  <img src="./chev-dow.svg" alt="" />
+                </button>
+              </div>
+              <!-- <div class="separator-line"></div> -->
+              <ul class="language-list">
+                <li>
+                  <img src="./javascript.svg" alt="JavaScript" />
+                  JavaScript
+                </li>
+                <li>
+                  <img src="./html.svg" alt="HTML" />
+                  HTML
+                </li>
+                <li>
+                  <img src="./css.svg" alt="CSS" />
+                  CSS
+                </li>
+              </ul>
+            </li>
+
+            <!-- /languages -->
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Frameworks</h3>
+                <button class="btn-chev" type="button">
+                  <img src="./chev-right.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+            </li>
+            <!-- /Frameworks -->
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Skills</h3>
+                <button class="btn-chev" type="button">
+                  <img src="./chev-right.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+            </li>
+            <!-- /Skills -->
+          </ul>
+        </section>
+        <section class="contact-me">
+          <h2>Contact me</h2>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form class="contact-form" action="#" method="post">
+            <input type="text" name="name" placeholder="Your Name" required />
+            <input
+              type="email"
+              name="email"
+              placeholder="Your Email"
+              required
+            />
+            <textarea
+              name="message"
+              cols="30"
+              rows="6"
+              placeholder="Write your message here"
+              required
+            ></textarea>
+            <button class="btn-submit" type="submit">Get In Touch</button>
+          </form>
+        </section>
       </main>
```

Comment:
- [ ] [Optional] You can do up to `About myself` section. You are assigned to do this contact form next week. So for now please remove it.